README.md

```diff
@@ -1,19 +1,20 @@
 # push-jobs cookbook
 
-Installs the Chef Push Jobs client package and sets it up to run as
+Installs the Chef Push client package and sets it up to run as
 a service.
 
 The official documentation is on
-[docs.getchef.com](http://docs.opscode.com/push_jobs.html)
+[docs.chef.io](http://docs.chef.io/push_jobs.html)
 
 # Requirements
 
-Requires Enterprise Chef with the Push
+Requires Chef Server with the Push
 Jobs feature.
 
 * Chef: 11.4.0 or higher
 * runit cookbook
 * windows cookbook
+* chef-ingredient cookbook
```

Comment:
It would be handy to link these to their respective supermarket pages.

* [runit](https://supermarket.chef.io/cookbooks/runit)
* [windows](https://supermarket.chef.io/cookbooks/windows)
* [chef-ingredient](https://supermarket.chef.io/cookbooks/chef-ingredient)